src/playlist.js

```diff
@@ -250,6 +251,51 @@ export const seekable = function(playlist, expired) {
     return createTimeRange(0, duration(playlist));
   }
 
+  segments = playlist.segments;
+  expiredSyncPoint = playlist.syncInfo;
+
+  // Find the first segment with timing information
+  for (let i = 0, l = segments.length; i < l; i++) {
+    let segment = segments[i];
+
+    if (typeof segment.start !== 'undefined') {
+      firstSyncPoint = {
+        mediaSequence: playlist.mediaSequence + i,
+        time: segment.start
+      };
+      break;
+    }
+  }
+
+  // We have no sync information for this playlist so we can't create a seekable range
+  if (!expiredSyncPoint && !firstSyncPoint) {
+    return createTimeRange();
+  }
+
+  // If we dont have an expired sync point, estimate expired time based on first sync,
```

Comment:
And it seems like the comment is mixed (order-wise) from the conditionals below.